full-stack-tests/presentation/src/frontend/UnifiedSelection.test.ts

```diff
@@ -185,6 +188,172 @@ describe("Unified Selection", () => {
+
+        await using<RegisteredRuleset, Promise<void>>(await Presentation.presentation.rulesets().add(groupByClassRuleset), async () => {
+          const rootNodes = await Presentation.presentation.getNodes({ imodel: groupingNodesIModel, rulesetOrId: groupByClassRuleset.id });
+          Presentation.selection.addToSelection("", groupingNodesIModel, [rootNodes[0].key]);
+          await waitForAllAsyncs([handler, Presentation.selection.getToolSelectionSyncHandler(groupingNodesIModel)!]);
+          expect(groupingNodesIModel.hilited.elements.size).to.be.equal(3);
+          objectIdArray.forEach((id) => expect(groupingNodesIModel.hilited.elements.hasId(id)).to.be.true);
+        });
+      });
+
+      it("hilites grouping nodes grouped by property", async () => {
+        const groupByPropertyRuleset: Ruleset = {
+          id: "groupByPropertyRuleset",
+          rules: [{
+            ruleType: RuleTypes.RootNodes,
+            specifications: [{
+              specType: ChildNodeSpecificationTypes.InstanceNodesOfSpecificClasses,
+              classes: { schemaName: "BisCore", classNames: [GeometricElement3d.className] },
+              arePolymorphic: true,
+              instanceFilter: "this.Parent = NULL",
+              groupByClass: false,
+              groupByLabel: false,
+            }],
+            customizationRules: [{
+              ruleType: RuleTypes.Grouping,
+              class: { schemaName: "BisCore", className: GeometricElement3d.className },
+              groups: [{
+                specType: GroupingSpecificationTypes.Property,
+                propertyName: "CodeValue",
+                createGroupForSingleItem: true,
+              }],
+            }],
+          }],
+        };
+
+        await using<RegisteredRuleset, Promise<void>>(await Presentation.presentation.rulesets().add(groupByPropertyRuleset), async () => {
+          const rootNodes = await Presentation.presentation.getNodes({ imodel: groupingNodesIModel, rulesetOrId: groupByPropertyRuleset.id });
+          Presentation.selection.addToSelection("", groupingNodesIModel, [rootNodes[0].key]);
+          await waitForAllAsyncs([handler, Presentation.selection.getToolSelectionSyncHandler(groupingNodesIModel)!]);
+          expect(groupingNodesIModel.hilited.elements.size).to.be.equal(2);
+          expect(groupingNodesIModel.hilited.elements.hasId(objectIdArray[0])).to.be.true;
+          expect(groupingNodesIModel.hilited.elements.hasId(objectIdArray[1])).to.be.true;
+
+          Presentation.selection.replaceSelection("", groupingNodesIModel, [rootNodes[1].key]);
+          await waitForAllAsyncs([handler, Presentation.selection.getToolSelectionSyncHandler(groupingNodesIModel)!]);
+          expect(groupingNodesIModel.hilited.elements.size).to.be.equal(1);
+          expect(groupingNodesIModel.hilited.elements.hasId(objectIdArray[2])).to.be.true;
+        });
+      });
+
+      it.skip("hilites grouping nodes grouped by label", async () => {
```

Comment:
please add a comment with a link to GH issue related to this `skip`

Reply:
Added.